### DISCUSSION TITLE
1 PCB to giveaway (USA), another to trade!

### DISCUSSION THREAD

**Discussion by HighMans** [sanni/cartreader#124] [General Chat]
Edit: Giveaway's over!

---

~~Hello all! Just ordered 5 PCBs from JLPCB and only need the three. They are printed with the latest PCB design as of 4/19/21 and have red solder mask with white silkscreen.~~

~~I'd like to give one away to a random person + free shipping, just need to enter with through google forms. (Github doesn't have PM's so e-mail's the best way to contact. Also shipping in USA only, sorry!) [Link to enter.](https://forms.gle/Z8kCtLcHNPL3a1Kk6)~~

~~The other I'd like to trade + PayPal for one or more the following: ~~PIC12F629(flashed with snesCIC)~~, MD Slot and SNES Slot.~~

~~Edit: Giveaway ends on 4/27/21 at 5pm EDT.~~

**Comment by Androxilogin** [comment]
I could program a chip and send it to you, however, I don't have any extra MD or SNES slots. I could probably get away with sending it in a regular envelope for free but I'll need your details. ~~[Redacted]~~ on Discord.
